lib/Doctrine/ORM/Configuration.php

```diff
@@ -70,6 +70,8 @@ public function setProxyDir($dir)
      * Gets the directory where Doctrine generates any necessary proxy class files.
      *
      * @return string|null
+     *
+     * @deprecated
```

Comment:
Why not also trigger a deprecation error?

Reply:
There is no replacement, it would be an error that cannot be muted/fixed, it would be just annoying without benefit.